seed/static/seed/js/services/matching_service.js

```diff
@@ -27,10 +27,10 @@ angular.module('BE.seed.service.matching', []).factory('matching_service', [
       });
     };
 
-    matching_service.mergeProperties = function (state_ids) {
+    matching_service.mergeProperties = function (property_view_ids) {
       spinner_utility.show();
-      return $http.post('/api/v2/properties/merge/', {
-        state_ids: state_ids
+      return $http.post('/api/v3/properties/merge/', {
+        property_view_ids
```

Comment:
I saw this with tax lots and was surprised this worked. We don't have to define the key for the `property_view_ids` value?

Reply:
I spent some time looking at the data, and though I don't totally understand where it comes from it seems to always have it 🤷‍♂️